src/test/java/io/confluent/connect/jdbc/dialect/SqliteDatabaseDialectTest.java

```diff
@@ -276,4 +280,10 @@ public void testDescribeTable() throws SQLException {
     assertEquals(false, columnDefn.isPrimaryKey());
     assertEquals(true, columnDefn.isOptional());
   }
+
+  @Test
+  public void useCurrentTimestampValue() throws SQLException {
+    Calendar cal = DateTimeUtils.getTimeZoneCalendar(TimeZone.getTimeZone("PST"));
+    assertNotNull(dialect.currentTimeOnDB(sqliteHelper.connection, cal));
```

Comment:
Do we want to verify that the returned string matches the format? Just checking that the method returns non-null doesn't seem too useful.

It's actually going to be difficult to verify the timestamp in the string, because the timestamp that the DB returns will be off by a fraction of a second of one that the test method gets, but that might be the difference between one minute and the next, or one hour and the next, or even one day and the next. So, about the best we could do is check that the format is correct and each component is not off by more than one (or seconds off by more than 5) from the test method's timestamp.

Reply:
That makes sense! I modified the unit test to support that verification now.